src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -32,13 +38,21 @@
 import io.flutter.console.FlutterConsoleHelper;
 import io.flutter.dart.DartPlugin;
 import io.flutter.utils.FlutterModuleUtils;
+import org.jdom.Element;
+import org.jdom.JDOMException;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
+import java.io.File;
+import java.io.IOException;
 import java.util.concurrent.atomic.AtomicBoolean;
 
 public class FlutterSdk {
   public static final String FLUTTER_SDK_GLOBAL_LIB_NAME = "Flutter SDK";
+
+  private static final String SDK_LIBRARY_PATH = ".idea/libraries/Dart_SDK.xml";
```

Comment:
`StartupActivity` has one very important documentation line :). The rest, I believe is pretty much clear from its name and method signature.
In general, I agree, a lot of API methods lack inline documentation. I used to learn by example.
Some useful docs for plugin writers are [here](http://www.jetbrains.org/intellij/sdk/docs/index.html).